package.json

```diff
@@ -60,6 +61,7 @@
   },
   "dependencies": {
     "grunt-shell": "^2.1.0",
+    "node-sass": "^4.5.3",
```

Comment:
its a really good idea to take all of the `^` out of here. You never know when updating a dependency will kill your app.